core/src/options/core.rs

```diff
@@ -51,6 +54,7 @@ impl Core {
             generics: di.generics.clone(),
             data: Data::try_empty_from(&di.data)?,
             default: Default::default(),
+            krate: None,
```

Comment:
```suggestion
            krate: Default::default(),
```

Let's keep consistency with other initializers.